README.md

```diff
@@ -32,6 +32,14 @@ bundle install
 
 See the 18F Pages documentation to learn how to [make a new page](https://pages.18f.gov/guides-template/add-a-new-page/) and [add it to the homepage navigation links](https://pages.18f.gov/guides-template/update-the-config-file/).
 
+### Fork or branch?
+
+Forking and branching are two ways of submitting pull requests to edit the Handbook.
+
+* **TTS teammates**: Please use ***branching*** to submit pull requests. Federalist Preview sites will only be built from a branch, and continuous integration can only succeed for PRs created from branches.
+
+* **External contributors**: Please use ***forking*** to submit pull requests, since non-TTS contributors do not have write access. Unfortunately, we won't be able to run Federalist Preview sites for your pull request; please build and serve the site locally to test instead.
+
```

Comment:
What do you think about linking to some relevant documentation on forking vs branching and/or how to do it the preferred way? Some of our handbook contributors are relatively unfamiliar with Git, and I'd like to pave the way for them as much as we can. 

I couldn't find a really clear and simple explanation, but I did think this one got fairly close. 

Also, I think some folks are forking or branching from directly in Github, or possibly from the github client. It may be worth reaching out to them (if we can figure out who they are) to see what their current workflow is, vs what we would recommend.

Reply:
@tbaxter-18f I think linking to relevant documentation is a great idea! Did you mean to share a link? I think you may have forgotten to paste it above? 

We could also add, "feel free to drop into `#tts-handbook` on Slack with any forking/branching questions"!

And I think the GitHub client question is a really good one! I'm going to ask about it on Slack now!

Reply:
I did neglect to share the link: https://blog.gitprime.com/the-definitive-guide-to-forks-and-branches-in-git/

I like the "you can always ask in slack" suggestion, too. I think the biggest thing is that we want to make it as simple and clear as possible to do the best thing, not have this become an a speed bump people have to get over on the way to contributing.

Reply:
@tbaxter-18f I love it and I think that's a great point!

Reply:
Side note on that link, not relevant to this PR: I think the "convergent" vs "divergent" framing is pretty interesting and may merit further thought at some point

Reply:
I tried using the GitHub web editor to edit a file, here's what that UI looks like:

<img width="957" alt="screen shot 2019-01-30 at 9 31 46 am" src="https://user-images.githubusercontent.com/3209501/51992137-13e29980-2472-11e9-8526-fe2e5feb1755.png">

There's no option to fork, just branch. I bet GitHub proposes you branch if you have write permissions (i.e. if you're a member of a TTS GitHub Org). If you don't have write permissions, I bet it suggests you either fork, or it doesn't offer the pencil icon at all. So the fork vs. branch question may not apply that much to folks who are using the GitHub web editor or desktop editor.

Reply:
Oooh, this answers the question about external contributors: GitHub will automatically propose a fork. https://help.github.com/articles/editing-files-in-another-user-s-repository/